Razmik/bash/comb_file/bash

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+x=$(find . -name 'math*')
+mkdir math
+
+mv math_file_1.txt math 
+mv math_gggg.txt math
```

Comment:
files from $x should be moved to the corresponding folder.
Will be merged with error